components/segment-bridge/base/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
we need to do something about the token for accessing this service account. AFAIK by default we would get a random token, but we need something a bit less random so we can can configure it for our client script and not have to replace it every time the cluster gets reprovisioned

Reply:
I'm looking up if we can somehow set this beforehand, maybe by setting a token secret? If not, will we need to run an oc command each time?

Reply:
As mentioned in previous channels, currently I couldn't find a way to pre-set the token, and I believe it is not doable. I've set a secret-access-token so the generated token will at least be long-lived.